server/src/http/routes/trainingLinks.controller.ts

```diff
@@ -0,0 +1,148 @@
+import { EligibleTrainingsForAppointment, FormationCatalogue } from "../../common/model/index.js"
+import express from "express"
+import { tryCatch } from "../middlewares/tryCatchMiddleware.js"
+import { asyncForEach } from "../../common/utils/asyncUtils.js"
+import { IFormationCatalogue } from "common/model/schema/formationCatalogue/formationCatalogue.types.js"
+import apiGeoAdresse from "../../common/utils/apiGeoAdresse.js"
+
+
+interface ITraining {
+    id: string
+    cle_ministere_educatif: string
+    mef: string
+    cfd: string
+    rncp: string
+    code_postal: string
+    uai: string
+}
+
+interface ILinks {
+    id: string
+    error: boolean
+    message: string
+    lien_prdv: string
+    lien_lba: string
+}
+const getPrdvLink = async (training: Partial<ITraining>): Promise<string> => {
+    if(!training.cle_ministere_educatif) {
+        return ""
+    } else {
+        return await EligibleTrainingsForAppointment.findOne({cle_ministere_educatif:training.cle_ministere_educatif}) ? `https://labonnealternance.apprentissage.beta.gouv.fr/espace-pro/form?referrer=lba&cleMinistereEducatif=${encodeURIComponent(training.cle_ministere_educatif)}`:""
```

Comment:
```suggestion
        return EligibleTrainingsForAppointment.findOne({cle_ministere_educatif:training.cle_ministere_educatif}) ? `https://labonnealternance.apprentissage.beta.gouv.fr/espace-pro/form?referrer=lba&cleMinistereEducatif=${encodeURIComponent(training.cle_ministere_educatif)}`:""
```

await pas necessaire quand tu return.